src/sdk/models/diagnostics.ts

```diff
@@ -3,3 +3,8 @@ export interface DiagnosticMetric  {
 	value: string;
 	type: string;
 }
+
+export interface DiagnosticFiddlerStatus {
```

Comment:
Looks like there's a similar interface [here](https://github.com/portalarchitects/skysync-cli/blob/5fcad3a564e291787e5a7dfd91de47169292d951/src/commands/diagnostics/fiddler/util.ts#L20)

Reply:
Removed the duplicate interface and referenced the exported SDK interface